driver/esp32/ili9XXX.py

```diff
@@ -537,7 +549,7 @@ class ili9488(ili9XXX):
     def __init__(self,
         miso=5, mosi=18, clk=19, cs=13, dc=12, rst=4, power=14, backlight=15, backlight_on=0, power_on=0,
         spihost=esp.HSPI_HOST, mhz=40, factor=8, hybrid=True, width=320, height=480,
-        colormode=COLOR_MODE_RGB, rot=PORTRAIT, invert=False, double_buffer=True, half_duplex=True,
+        colormode=COLOR_MODE_RGB, rot=180, invert=False, double_buffer=True, half_duplex=True,
```

Comment:
`PORTRAIT` is currently defined to `MADCTL_MX` which is `const(0x40)`.  
Why did you change it to 180?

Reply:
That was a mistake and has been rectified

Reply:
Are you sure you committed and pushed?  
I can still see `rot=180` on ili9488

Reply:
I will check it on Monday when I get back